core/app/controllers/Application.java

```diff
@@ -49,7 +49,7 @@ public Application(ActorSystem system) {
         FrontEndDBInterface f = new FrontEndDBInterface();
         List<models.Configuration> configList;
         try {
-            configList = f.getConfigList();
+            configList = f.getConfigList("Dev");
```

Comment:
We need to give the team name to this function now, so we're only displaying the configurations of that team.